File: Mobile/lib/features/alerts/alert_model.dart
class AlertModel {
  final String id;
  final String userName;
  final String alertType;
  final DateTime createdAt;
  final bool isHandled;
  final String location;
  final String message;

  AlertModel({
    required this.id,
    required this.userName,
    required this.alertType,
    required this.createdAt,
    this.isHandled = false,
    this.location = '',
    this.message = '',
  });
}

File: backend/Models/AlertModel.cs
using System;

namespace ve_si_ao_api.Models
{
    public class AlertModel
    {
        public int Id { get; set; }
        public string UserName { get; set; } = string.Empty;
        public string AlertType { get; set; } = "SOS"; 
        public DateTime CreatedAt { get; set; } = DateTime.Now;
        public bool IsHandled { get; set; } = false;
        public string Location { get; set; } = string.Empty;
        public string Message { get; set; } = string.Empty;
    }
}
